assets/js/components/HostsList.jsx

```diff
@@ -166,7 +166,10 @@ function HostsList() {
       provider: host.provider,
       sid: sapSystemList.map((sapSystem) => sapSystem.sid),
       cluster,
-      agent_version: host.agent_version,
+      agent_version:
+        host.agent_version.length > 15
+          ? `${host.agent_version.slice(0, 15)}...`
+          : host.agent_version,
```

Comment:
You can use a `text-overflow: ellipsis` to do this, no need to act at this level

Reply:
I had actually tried this before going ahead with changing it in the .jsx but it didn't have any effect. I can see the style being applied to the span but the full string is still displayed. I suspect that the parent's have something preventing the overflow.

If this is a big smell I'll give it another go.

Reply:
Nevermind this, I managed to make it work with `text-overflow: ellipsis` :+1: